apps/files_external/src/actions/inlineStorageCheckAction.ts

```diff
@@ -47,47 +30,55 @@ export const action = new FileAction({
 	/**
 	 * Use this function to check the storage availability
 	 * We then update the node attributes directly.
+	 *
+	 * @param node The node to render inline
 	 */
 	async renderInline(node: Node) {
-		let config = null as any as StorageConfig
-		try {
-			const response = await getStatus(node.attributes.id, node.attributes.scope === 'system')
-			config = response.data
-			Vue.set(node.attributes, 'config', config)
+		const span = document.createElement('span')
+		span.className = 'files-list__row-status'
+		span.innerHTML = t('files_external', 'Checking storage...')
```

Comment:
```suggestion
		span.innerHTML = t('files_external', 'Checking storage …')
```

- https://github.com/nextcloud/server/pull/50946